src/pybind/mgr/dashboard/frontend/src/app/shared/services/notification.service.ts

```diff
@@ -146,4 +152,12 @@ export class NotificationService {
   cancel(timeoutId) {
     window.clearTimeout(timeoutId);
   }
+
+  /**
+   * Suspend showing the notification toasties.
+   * @param {boolean} suspend Set to ``true`` to do not show toasties.
```

Comment:
`Set to ``true`` to do not show toasties.` => `Set to ``true`` to not show toasties` or `Set to ``true`` to disable/hide toasties` (which I prefer)